ci/testOnCommit.groovy

```diff
@@ -0,0 +1,26 @@
+pipeline {
+    agent {
```

Comment:
could you try to add required Job parameter.


    environment {
        CONFIG= "
environ:
  # Branch settings
  CORE_BRANCH: ${env.MEDIACOMMIT}
  PRODUCT_VERSION: 2023.1
  OV_SOURCE_BRANCH: master
  OV_CONTRIB_BRANCH: master
  OV_SOURCE_ORG: openvinotoolkit
  OV_CONTRIB_ORG: openvinotoolkit
  OV_USE_BINARY: 1
  # Compilation settings
  #NO_DOCKER_CACHE: true

  # Repo settings
  TT_OVMS_C_URL: https://github.com/openvinotoolkit/mediapipe
  TT_OV_REPO_PATH: https://github.com/openvinotoolkit/openvino
  # Advanced validation settings
  TT_JENKINS_JOB_NAME: '"master_${config.environ.PRODUCT_VERSION}_${ovms_commit_id}_OV${ov_commit_id}"'
  TT_OVMS_C_REPO_PATH: "${WORKSPACE}/openvino-mediapipe/"
  TT_OVMS_TEST_SIMPLE_JOB: "OpenVinoMediapipeWorker"    # Name of Jenkins job to be executed on worker
  PIPELINE_PATH: '"/usr/local/ovms-test/ci/agileTest.groovy"'
  TT_OVMS_TEST_COMMAND: 'ovms-test'
  OVMS_TEST_PACKAGE: http://s3.toolbox.iotg.sclab.intel.com/test/install_packages/ovms-test_deb/ovms-test-latest.deb
builds:
  - name: ubuntu-from-mediapipe
    agent: builder04
    environ:
      BASE_OS: ubuntu
      BASE_IMAGE: ubuntu:20.04
      OV_USE_BINARY: 0       "
}

Reply:
Please do not change requirements after implementation, after this PR is merged everyone can change parameters passed to target build OpenVinoMediapipe (see 'MEDIACOMMIT' example parameter). Above parameter is not properly defined.